Documents/DevDoc.md

```diff
@@ -1 +1,61 @@
-# DevDoc
+# Welcome to OAPS-Jifry
+
+**OAPS-Jifry** is a re-implementation of [OAPS](https://github.com/youmowang/zjnuservice) by Yun Dabang, and Ni Zhongjie, as OOAD Fall 2018 course project, using [JSP](https://en.wikipedia.org/wiki/JavaServer_Pages) technology.
+
+
+# How to contribute
+
+Currently, this project is maintained by team **MIJC** (Mahdi,IBRAHIM,Jifry, and Clive) and everyone else is welcomed to join us and help us to improve the project by any means.
+
+## Add Your Information to the Contributor List
+
+Clone the main repository:
+
+    $git clone https://github.com/lanlab-org/OAPS-Jifry
+Edit the **README.md** file, append your:
+
+> github account - Name - Student ID
+
+to the contributor list.
+
+Then, add your changes to the master repository, 
+
+    $git add .
+commit the changes, 
+
+    $git commit -m "your commit message"
+   and push them to the master repository.
+
+    $git push origin master
+## Development Plan with KanBan
+For more information about the project's maintenance progress and to get a visual representation of the whole plan, see  [the project board on Kanboard](http://118.25.96.118/kanboard/?controller=BoardViewController&action=readonly&token=80ecf8f431a8dfafbc437dbc621b98d1c37c02790fd89f9cedc49a4c8d36).
+If you would like to join us on KanBoard, contact (lanhui at zjnu.edu.cn) to create an account for you, then contact (1525200991 at qq.com) to invite you to our project borad.
+## Which Area in the Project to Contribute to
+
+If you wish to contribute to the source code see section **"Project Environment"**.
+However, if you wish to contribute to the project's documentation, homepage, etc, of the non-source-code-related areas. You can skip **"Project Environment"** section.
```

Comment:
@MarshmallowClive 

if you wish to contribute to non-source-code-related areas, e.g.,   documentation, homepage, etc, you can skip that section.

Hui